### PR TITLE
[CARBONDATA-2050] Add example of query data with specified segments

### PR DESCRIPTION
**Pull request by chenliang613**
Add example of query data with specified segments and also consider compaction sceanrios.

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [X] Any interfaces changed?
 NA
 - [X] Any backward compatibility impacted?
 NA
 - [X] Document update required?
YES
 - [X] Testing done
YES
       
 - [X] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA. 
NA

### CODE REVIEW

File: docs/data-management-on-carbondata.md [hunk 1]
```diff
@@ -781,59 +781,39 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
...
   ```
   SHOW SEGMENTS FOR TABLE [db_name.]table_name LIMIT number_of_segments
   ```
   
-  Set the segment IDs
-  
+  Set the segment IDs for table
   ```
-  SET cabon.input.segments.<database_name>.<table_name> = <list of segment IDs>;
+  SET cabon.input.segments.<database_name>.<table_name> = <list of segment IDs>
```
Comment (QiangCai) [inline docs/data-management-on-carbondata.md]: cabon => carbon

Comment (QiangCai) [reply docs/data-management-on-carbondata.md]: please fix all typo in this file.

**Comment by QiangCai** [comment]
LGTM

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/1706/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/2941/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/2972/

**Pull request closed without merging**